EventSubscriber/BeforeMessageProcessingSubscriber.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auxmoney\OpentracingAmqplibRabbitMqBundle\EventSubscriber;
+
+use Auxmoney\OpentracingAmqplibRabbitMqBundle\Consumer\Consumer;
+use Auxmoney\OpentracingBundle\Internal\Utility;
+use Auxmoney\OpentracingBundle\Service\Tracing;
+use OldSound\RabbitMqBundle\Event\AMQPEvent;
+use OldSound\RabbitMqBundle\Event\BeforeProcessingMessageEvent;
+use OpenTracing\Reference;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use const OpenTracing\Tags\SPAN_KIND;
+use const OpenTracing\Tags\SPAN_KIND_MESSAGE_BUS_CONSUMER;
+
+final class BeforeMessageProcessingSubscriber implements EventSubscriberInterface
+{
+    private const SPAN_NAME = 'Processing message from "%s" queue';
```

Comment:
Please add a plugin prefix, like `RabbitMq: `, so that you can see the plugin in the collapsed jaeger trace view.